home/choices.py

```diff
@@ -0,0 +1,37 @@
+RACE_CHOICES = (('Politely Decline', 'Politely Decline'),
+ ('American Indian/Alaskan Native', 'American Indian/Alaskan Native'),
+ ('Asian', 'Asian'),
+ ('Black/African American', 'Black/African American'),
+ ('Native Hawaiian/Other Pacific Islander', 'Native Hawaiian/Other Pacific Islander'),
+ ('White', 'White'),
+ ('American Indian/Alaskan Native AND Black/African American', 'American Indian/Alaskan Native AND Black/African American'),
+ ('American Indian/Alaskan Native AND White', 'American Indian/Alaskan Native AND White'),
+ ('Asian AND White', 'Asian AND White'),
+ ('Black/African American AND White', 'Black/African American AND White'),
+ ('Other/Multiracial', 'Other/Multiracial'))
+
+ETHNICITY_CHOICES = (('Politely Decline', 'Politely Decline'),
+ ('Hispanic/Latinx', 'Hispanic/Latinx'),
+ ('Not Hispanic/Latinx', 'Not Hispanic/Latinx'))
+
+GENDER_CHOICES = (('Politely Decline', 'Politely Decline'),
+ ('Female', 'Female'),
+ ('Male', 'Male'),
+ ('Genderqueer/Gender Non-binary', 'Genderqueer/Gender Non-binary'),
+ ('Trans Female', 'Trans Female'),
+ ('Trans Male', 'Trans Male'), ('Other', 'Not Listed'))
```

Comment:
This mirrors Contact->Gender choices on SFDB.  I could change using Object Manager in the sandbox but we'd then have to make changes in the larger database as well.  I think we should avoid making changes that we don't absolutely have to in the main DB to avoid unforseen consequence cascades.

![image](https://user-images.githubusercontent.com/37964808/60215018-9dc54180-981b-11e9-8256-637ea394a968.png)